polkadot-wiki/docusaurus.config.js

```diff
@@ -195,7 +195,7 @@ module.exports = {
           },
           {
             to: '/docs/learn-transactions',
-            from: ['/docs/learn-transactions-index', '/docs/learn-extrinsics', '/docs/learn-balance-transfers']
+            from: ['/docs/learn-transactions-index', '/docs/learn-extrinsics', '/docs/learn-balance-transfers', '/docs/learn-transaction-fees']
```

Comment:
I wonder if we could have a more specific redirect, i.e.,: `/docs/learn-transaction-fees`->  `/docs/learn-transactions#transaction-fees`

Reply:
hey Bader I remember doing this and crashed the Prod deployment :D